packages/currencies/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ledgerhq/currencies",
-  "version": "4.5.0",
+  "version": "4.6.0",
```

Comment:
we don't bump the version in PRs , it is done automatically by lerna when we are publishing from master